compiler/main.bal

```diff
@@ -47,12 +48,20 @@ public function main(string[] filenames, *Options opts) returns error? {
         debugLevel: check nback:validDebugLevel(opts.debugLevel)
     };
     int errorFileCount = 0;
+    d:Printer dPrinter;
+    if opts.errorReport {
+        dPrinter = new d:HTMLPrinter("errorLog.html");
```

Comment:
If you are writing to a file (not stderr/stdout), then the option needs to be specify the filename (and so is string-valued).  Call is something like `htmlError`.

Reply:
fixed